wiki/Help_Center/en.md

```diff
@@ -316,153 +323,215 @@ If you wish to prevent anyone not on your friendslist from sending you private m
 
-If your supporter tag runs out, your user page will remain visible and you will still be able to edit it.
+Make sure to include any chatlogs or other evidence in the email, and our support staff will look into your case and see it sorted for you.
 
-### When I try to register, all the fields say "invalid request"!
+### What are these "user pages" that I see on other players profiles?
 
-This is a known issue, and we aren't yet sure what causes it. Some possible workarounds are:
+**User pages are special profile elements that all osu!supporters get access to. They can contain whatever you like - pictures of your achievements, your beatmaps, basically anything you wish!**
 
-- Make sure JavaScript is enabled and osu.ppy.sh is allowed to use it
-- Register on another browser
-- Registering on a different computer or Internet connection
+osu!supporters are given the privilege of editing their very own page for display on their profile for the world to see. User pages may contain anything you like - so long as it follows the [community rules](/wiki/Rules)!
 
-If none of these solutions work for you, or the first two don't work and you have no access to another computer or internet connection, please contact support by sending an e-mail to [accounts@ppy.sh](mailto://accounts@ppy.sh) and we'll try to get it resolved ASAP.
+Even if your supporter tag happens to run out, your user page will remain visible and you will still be able to edit its contents.
 
 ### Do I need to use my email address to register?
 
-No - ***osu!* authenticates your account by having you log in to the game client.** If you have trouble connecting with the client, please check your firewall and anti-virus settings to make sure osu! isn't being blocked.
+**No, you don't. We authenticate your account by having you log in via the game client, so an email is not necessary during the registration process.**
+
+If you have trouble connecting via the game client, please check your firewall and anti-virus settings to make sure osu! isn't being blocked.
+
+While having an email isn't required, we do recommend that you associate one with your account, as if you ever lose your password or cannot access your account, your registration email is our first point of contact with you.
 
-We highly recommend using an email address that you have regular access to (and is not a disposable 24hr "fake" email), as you will need to use the email to recover your password should you lose it, and also to talk with support representatives.
+### Why can't I search for a beatmap artist who's name is less than 3 characters long?
 
-### I can't search for a beatmap artist who's name is less than 3 characters long!
+**Our local search engine can't handle this due to technical limitations, but you can still search using the Google Custom Search bar at the top left of most pages on the site.**
```

Comment:
this may not be a thing for the new site... but you could use Google itself to do the search... using something like `xi site:osu.ppy.sh` 🤷‍♀️